### PR TITLE
[Order] added order additionalInformation

### PR DESCRIPTION
**Pull request by starspire**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | 
| License       | MIT
| Doc PR        | 

Added posibility for user to input type some text (like some information about his order to the seller) before checkout this cart. Handled in finalize table and current orders. I'm not sure if the parameter name is correct so any suggestion are higly appreciated.

### CODE REVIEW

**Comment by michalmarcinkowski** [comment]
Maybe `additional information` will be more appropriate than `details`, WDYT?

**Comment by tristanbes** [comment]
i'm also :+1: with `additionnal_information`

**Comment by Zales0123** [comment]
:+1: ``details`` can be misunderstood, ``additional information`` is much more clear.

**Comment by pjedrzejewski** [comment]
:+1: for ``additionalInformation`` field, this is what we used for this field in most of projects. :P

File: app/migrations/Version20160112125003.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('DROP INDEX fulltext_search_idx ON sylius_search_index');
```
Comment (pjedrzejewski) [inline app/migrations/Version20160112125003.php]: This hsould not be here. :)

**Comment by starspire** [comment]
Parameter changed to additionalInformation :)

**Comment by pjedrzejewski** [comment]
@starspire Could you rebase, please? Thanks! I will merge it then.

**Comment by starspire** [comment]
@pjedrzejwski Done.

**Comment by pjedrzejewski** [comment]
Thank you Krzysztof! :+1:

**Pull request merged**